component styles/Avatar.css

```diff
@@ -0,0 +1,52 @@
+.avatar-container{
+    display: flex;
+    width: 50%;
+    height: max-content;
```

Comment:
Not sure if this property is required. Kindly comment and check.

Reply:
Actually there is a page-container which is parent for all components and then there are separate containers for each component. I thought it will be flexible in future.